sanity-studio/schemas/time-range.js

```diff
@@ -0,0 +1,19 @@
+export default {
+  title: 'Time Range',
+  name: 'timeRange',
+  type: 'object',
+  fields: [
+    {
+      title: 'Hour',
+      name: 'hour',
+      type: 'number',
+      validation: Rule => Rule.min(1).max(12).precision(2).positive().integer(),
```

Comment:
Do we need AM/PM information along with hours and minutes?

Reply:
good point. I should include am/pm information in the time schema rather than Weekly schedule schema.